src/test/java/io/github/genomicdatainfrastructure/daam/mappers/RemsApplicationMapperTest.java

```diff
@@ -0,0 +1,165 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package io.github.genomicdatainfrastructure.daam.mappers;
+
+import io.github.genomicdatainfrastructure.daam.model.*;
+import io.github.genomicdatainfrastructure.daam.remote.rems.model.*;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RemsApplicationMapperTest {
+
+    private static final ApplicationMapper mapper = new RemsApplicationMapper();
+
+    @Test
+    public void should_map_correctly_complete_rems_application_to_retrieved_application() {
+        RetrievedApplication retrievedApplication = mapper.from(createApplication());
+
+        assertEquals(1L, retrievedApplication.getId());
```

Comment:
suggestion: Perhaps you could use `AssertJ` to improve the readability of this test.